oci/namespaces.go

```diff
@@ -4,13 +4,9 @@ import specs "github.com/opencontainers/runtime-spec/specs-go"
 
 // RemoveNamespace removes the `nsType` namespace from OCI spec `s`
 func RemoveNamespace(s *specs.Spec, nsType specs.NamespaceType) {
-	idx := -1
 	for i, n := range s.Linux.Namespaces {
 		if n.Type == nsType {
-			idx = i
+			s.Linux.Namespaces = append(s.Linux.Namespaces[:i], s.Linux.Namespaces[i+1:]...)
```

Comment:
Is this the same though? The old version only removed the last occurrence of `nsType`, whereas the new version removes every occurrence; https://play.golang.org/p/PzvDLUCEeI